ENIGMAsystem/SHELL/Universal_System/estring.h

```diff
@@ -22,6 +22,20 @@
 
 #include <string>
 
+#include "../../../CompilerSource/OS_Switchboard.h"
+
+#if CURRENT_PLATFORM_ID == OS_WINDOWS
+
+#include <windows.h>
+#include <wchar.h>
+#include <vector>
```

Comment:
You aren't using `vector`, here, and this can cause problems in the current function parser. There's technically no sense including `windows.h` for this; we know our implementation supports `wchar_t`, but since this is Windows-only for now, I guess it's fine.

Reply:
Ok, I removed windows.h and vector